skins/oasis/js/Tracking.js

```diff
@@ -219,7 +219,7 @@ jQuery(function ($) {
 
 	/** contribute **/
 
-	$wikiHeader.find('.buttons .contribute').on('mousedown', 'a', function (event) {
+	$wikiHeader.find('.buttons .contribute').add('#WikiaPageHeader').on('mousedown', 'a', function (event) {
```

Comment:
In the callback we filter the links using `data-id` attribute. Also, this is the same selector as in https://github.com/Wikia/app/blob/3ccf876cbf2cce90734ab37d23742a5ede58ccff/skins/oasis/js/Tracking.js#L73